### ISSUE TITLE
Use dashes instead of minus signs

### ISSUE THREAD

**Issue by wldhx** [goto-ru/paulgraham-college#8]

**Comment by danya02** [comment]
I think a simple string substitution program might do the trick. However, I need an example of a dash to do the substitution. Could you please send at least one dash in a comment?

**Comment by wldhx** [comment]
Here's one: `–`.

For the record, that's an en dash; we could also use an em one (`—`), but I _think_ en might be more conventional; not sure though. If you'll be taking this issue, I'll leave the choice to your discretion.

**Comment by danya02** [comment]
No, I think the en dash is better, because it is more like the minus sign, so it seems it has a higher chance of being interpreted by text-only terminals.

**Comment by wldhx** [comment]
I believe ASCII includes both, so that should not be of concern.

**Comment by danya02** [comment]
Done, in a pull request #10. Managed to do it with one line of Bash code:

``` sh
danya@danya-master-desktop:~/paulgraham-college$ sed -e "y/-/–/" <translation.rst >translation1.rst && dd if=translation1.rst of=translation.rst && rm translation1.rst 
```

**Comment by wldhx** [comment]
You could do it simpler, actually: `sed s/-/–/g translation.rst -i`

**Comment by danya02** [comment]
Maybe, but I used the copying to another file because I didn't know about the `-i` key, and I used the `y` command since a line might contain more than one minus. Thanks for pointing it out, though.

**Comment by wldhx** [comment]
re: one minus, That's what the `g` here is for :) Though `y` vs `s/g` seems irrelevant here anyway. Also, FYI `sponge` can be used to replace this IO construction.

**Comment by danya02** [comment]
Right :). It's just that I don't really know many functions of `sed`, and any other stream editing programs for that matter. Will try to find more of those, and THX 1.0E6 for the new info.

**Comment by danya02** [comment]
I thought this issue was closed by the merge of #10?

**Comment by wldhx** [comment]
Right. /me should have added "Closes #8" in the #10 commit notes.
